ui/components/app/name/name-details/name-details.tsx

```diff
@@ -381,7 +452,11 @@ export default function NameDetails({
                 hideDropdownIfNoOptions
                 value={name}
                 options={proposedNameOptions}
-                placeholder={t('nameSetPlaceholder')}
+                placeholder={
+                  trustLabel
+                    ? `Suggested: ${trustLabel}`
+                    : t('nameSetPlaceholder')
+                }
```

Comment:
Just confirming that we want to start displaying the trust label in the placeholder.. above anything else?